packages/dds/tree/src/feature-libraries/chunked-forest/chunkedForest.ts

```diff
@@ -150,7 +151,11 @@ export class ChunkedForest implements IEditableForest {
 			attach(source: FieldKey, count: number, destination: PlaceIndex): void {
 				this.attachEdit(source, count, destination);
 			},
-			detach(source: Range, destination: FieldKey): void {
+			detach(
+				source: Range,
+				destination: FieldKey,
+				destinationDetachedNodeId: DeltaDetachedNodeId,
```

Comment:
I personally wouldn't have bothered with these changes since the parameters aren't being used, but I do think it's an interesting question whether it's best to list the unused params or not, it very well might be. I started a conversation on Teams to see what the rest of the team thinks.

Reply:
The team seems to lean towards listing them out. Resolving conversation.